### PR TITLE
[Console] Add placeholder formatters per ProgressBar instance

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Resolves https://github.com/symfony/symfony-docs/issues/17545

### CODE REVIEW

File: components/console/helpers/progressbar.rst [hunk 1]
```diff
@@ -337,13 +337,23 @@ display that are not available in the list of built-in placeholders, you can
...
 
+It is also possible to set a placeholder formatter per ProgressBar instance
+with the ``setPlaceholderFormatter`` method::
+
+    $progressBar = new ProgressBar($output, 3, 0);
+    $progressBar->setFormat('%countdown% [%bar%]');
+    $progressBar->setPlaceholderFormatter('countdown', function (ProgressBar $progressBar) {
+        return $progressBar->getMaxSteps() - $progressBar->getProgress();
+    });
+
```
Comment (alamirault) [inline components/console/helpers/progressbar.rst:356]: We should probably add

```
.. versionadded:: 6.3

    The ``setPlaceholderFormatter()`` method was introduced in Symfony 6.3.
```

Comment (alexandre-daubois) [reply components/console/helpers/progressbar.rst:356]: I thought of this, but wasn't sure! Do you know if there is any "rule" to it, when to add it or not?

Comment (javiereguiluz) [reply components/console/helpers/progressbar.rst:356]: @alexandre-daubois we add the `versionadded` directive for any new option, method, class, feature, etc. added to Symfony. We also have the `deprecated` directive to do the opposite.

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Alexandre.

**Comment by GromNaN** [comment]
Thank you.

**Pull request merged**